documentation/resource_mariadb_client_install.md

```diff
@@ -0,0 +1,24 @@
+# mariadb_client_install
+
+This resource installs mariadb client packages.
+
+## Actions
+
+- `install` - (default) Install client packages
+
+## Properties
+
+Name                | Types             | Description                                                   | Default                                   | Required?
+------------------- | ----------------- | ------------------------------------------------------------- | ----------------------------------------- | ---------
+`version`           | String            | Version of MariaDB to install                                 | '10.3'                                    | no
```

Comment:
```suggestion
`version`           | String            | Version of MariaDB to install                                 | `10.3`                                    | no
```

Reply:
I think this stlye *might look better when it's parsed.

Reply:
ok will adapt the whole documentation :-D